content/en/docs/contributing/release-process.md

```diff
@@ -102,34 +128,54 @@ The process for cutting a minor release is as follows:
 
         You can view the progress by clicking the Google Cloud Build URL in the output of this command.
 
-   2. Next publish the release artifacts for real.
+   2. While the build is running, send a third Slack message in reply to
+      the first message:
 
-        If the last step succeeded, you can now re-run the `cmrel publish` with the `--nomock` argument to actually publish the release artifacts to GitHub, `Quay.io`, [Helm Hub] etc.
+      > Publish dry-run step: <https://console.cloud.google.com/cloud-build/builds/16f6f875-0a23-4fff-b24d-3de0af207463?project=1021342095237>
 
-       ```bash
-       cmrel publish --nomock --release-name <RELEASE_NAME>
-       ```
+   3. Next publish the release artifacts for real.
+
+      If the last step succeeded, you can now re-run the `cmrel publish`
+      with the `--nomock` argument to actually publish the release
+      artifacts to GitHub, `Quay.io`, [Helm Hub] etc.
+
+      ```bash
+      cmrel publish --nomock --release-name <RELEASE_NAME>
+      ```
+
+      NOTE: At this stage there will be a draft release on GitHub and a
+      live release on [Helm Hub]. So you must now complete the release
+      process quickly otherwise users of the latest release on[Helm Hub]
+      will encounter errors, because the manual CRD install URL will not be
+      available yet.
 
-        NOTE: At this stage there will be a draft release on GitHub and a live release on [Helm Hub].
-        So you must now complete the release process quickly otherwise users of the latest release on [Helm Hub] will encounter errors,
-        because the manual CRD install URL will not be available yet.
+   4. While the build is running, send a fourth Slack message in reply to
+      the first message:
+
+      > Publish step: <https://console.cloud.google.com/cloud-build/builds/b6fef12b-2e81-4486-9f1f-d00592351789?project=1021342095237>
 
 7. Publish the GitHub release
 
    1. Visit the draft GitHub release and paste in the release notes that you generated earlier.
 
-        You will need to manually edit the content to match the style of earlier releases.
-        In particular:
+      You will need to manually edit the content to match the style of earlier releases.
+      In particular:
 
-       * Remove package related changes
-       * Replace links to `@jetstack-bot`, in cherry-pick PRs, with links to the GitHub handle of the author of the original PR.
+      * Remove package related changes
+      * Replace links to `@jetstack-bot`, in cherry-pick PRs, with links to
+        the GitHub handle of the author of the original PR.
```

Comment:
This didn't seem to happen with the latest version of `krel release-notes` -- the release notes had the correct authors -- so perhaps we can drop this line.